sources-grammars.ts

```diff
@@ -1107,6 +1107,7 @@ export const sourcesMarketplace: GrammarSource[] = [
   {
     name: 'mermaid',
     displayName: 'Mermaid',
+    aliases: ['mmd', 'mermaid-example'],
```

Comment:
I am not sure about `mermaid-example`, why it's needed?

Reply:
Is the default one used in mermaid docs. 
But I can ask them to change

Reply:
@antfu , not sure I should close the conversation or just wait you approval.😁